packages/paste-core/components/progress-steps/src/ProgressStepSeparator.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+import type {BoxProps} from '@twilio-paste/box';
+
+import {ProgressStepsContext} from './ProgressStepsContext';
+
+const VerticalStyles: BoxProps = {
+  borderLeftWidth: 'borderWidth20',
+  borderLeftStyle: 'solid',
+  borderLeftColor: 'colorBorderWeak',
```

Comment:
chore: sorry, oversight on the design. Let's change this to colorBorderWeaker to match the Separator component.